### ISSUE TITLE
In summary.txt what's the difference between length and bases?

### ISSUE THREAD

**Issue by nuno5645** [brentp/mosdepth#188]
chrom	         length	        bases	     mean	     min	max
chr13	         115169878	8508921	     0.07	      0	8833
chr13_region	 13390	        10009503     747.54      0	8833
chr17	         81195210	5713210	     0.07	      0	8689
chr17_region	 8316	        6078518	     730.94      0	8689
total	                 196365088     14222131     0.07	      0	8833
total_region	 21706	        16088021    741.18	      0	8833

so basically i want to get the number of bases and mapped bases so i can add to my quality metrics report.
The problem is that i am at a loss at which variables i should look at
